### PR TITLE
Allow showing AST and control flow graphs

### PR DESCRIPTION
**Pull request by Ellpeck**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,18 @@
   "main": "./out/extension.js",
   "contributes": {
     "commands": [
+      {
+        "command": "vscode-flowr.ast",
+        "title": "Show AST",
```
Comment (EagleoutIce) [inline package.json:35]:
```suggestion
        "title": "Show Abstract Syntax Tree of the File",
```

File: package.json [hunk 2]
```diff
@@ -30,6 +30,18 @@
...
+        "icon": "$(graph)",
+        "category": "flowR"
+      },
+      {
+        "command": "vscode-flowr.cfg",
+        "title": "Show Control Flow Graph",
```
Comment (EagleoutIce) [inline package.json:41]:
```suggestion
        "title": "Show Control Flow Graph of the File",
```

File: src/flowr/internal-session.ts [hunk 3]
```diff
@@ -110,6 +113,30 @@ export class FlowrInternalSession implements FlowrSession {
 		return dataflowGraphToMermaid(result.dataflow, result.normalize.idMap)
 	}
 
+	async retrieveAstMermaid(document: vscode.TextDocument): Promise<string> {
+		if(!this.shell) {
+			return ''
+		}
+		const result = await new SteppingSlicer({
```
Comment (EagleoutIce) [inline src/flowr/internal-session.ts:120]: We should schedule a migration issue for this, once df2 is released as 2.0.0 of flowr

Comment (Ellpeck) [reply src/flowr/internal-session.ts:120]: #84

**Pull request merged**